lib/build.js

```diff
@@ -15,25 +36,25 @@ function getBabelTarget(envConfig) {
   return unknown ? '8.15.0' : current.replace(/^nodejs/, '');
 }
 
-function haveBabelrc(functionsDir) {
-  const cwd = process.cwd();
-
-  return (
-    fs.existsSync(path.join(cwd, '.babelrc')) ||
-    functionsDir.split('/').some((dir) => {
-      const indexOf = functionsDir.indexOf(dir);
-      const dirToSearch = functionsDir.substr(0, indexOf);
-
-      return fs.existsSync(path.join(cwd, dirToSearch, '.babelrc'));
-    })
+function existsBabelConfig(functionsDir) {
+  const babelConfigFile = findUp.sync(
+    [...BABEL_ROOT_CONFIG_FILENAMES, ...BABEL_RELATIVE_CONFIG_FILENAMES],
+    {
+      cwd: path.join(process.cwd(), functionsDir),
+    },
   );
+  if (babelConfigFile) {
+    // Ensure babel config files outside of working dir are not accounted for
+    return babelConfigFile.includes(process.cwd());
```

Comment:
Will this work for monorepos when the babel config is in the root directory?
We could `find-up` a `.git` and use that value (or `cwd` if it doesn't exist).

We might also want to user a matcher and [`findUp.stop`](https://github.com/sindresorhus/find-up#findupstop) to stop the search if we detect we're outside the root.

Reply:
I don't think it would make sense to search outside the cwd. Babel will by default [also not look outside the cwd](https://babeljs.io/docs/en/config-files#project-wide-configuration) for a config file.

Do you imagine a scenario in a monorepo where the user runs `netlify-lambda` from a subdir while they want to use the babel config in the root dir?